src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,35 +58,44 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${id}`}
```

Comment:
This will have the same effect, but easier to read
```suggestion
              Todo #{id}
```